plugin/src/App/Components/IconButton.lua

```diff
@@ -75,6 +75,8 @@ function IconButton:render()
 
 			BackgroundTransparency = 1,
 		}),
+
+		Children = e("Folder", {}, self.props[Roact.Children]),
```

Comment:
Can we use a fragment here instead of a folder?

Reply:
I chose Folder because I found it helpful in debugging as it creates a clear distinction between "components that are children" and "children that are just part of the button component".

Reply:
Given that there aren't many instances or components involved, the extra folder usually doesn't help that much. We we don't spend that much time debugging the individual instances in buttons, so always adding a folder when we don't have to might not be a win.

There have been bugs historically from introducing folders in places like this. That might not be the case anymore, but also gives me pause. That was one reason for Roact gaining fragments when it did, a limitation specifically with ScreenGui at the time!